src/pages/blog/[...slug].js

```diff
@@ -1,15 +1,15 @@
-import matter from 'gray-matter';
-import glob from 'glob';
-import { fetchGitHubUser } from '@/utils/fetchData';
+import matter from "gray-matter";
+import glob from "glob";
+import { fetchGitHubUser } from "@/utils/fetchData";
```

Comment:
Why change this to double quotes?

Reply:
just an accidental side effect of format on save with Zed using prettier defaults since there isn't a prettierconfig specified in the project, I'll run the whole thing through a set format at the end just to make it consistent